taskcluster/app_services_taskgraph/build_config.py

```diff
@@ -10,20 +10,45 @@
 from taskgraph.util.memoize import memoize
 
 
+EXTENSIONS = {
+    'aar': ('.aar', '.pom', '-sources.jar'),
+    'jar': ('.jar', '.pom')
+
+}
+CHECKSUMS_EXTENSIONS = ('.sha1', '.md5')
+
+
 def get_components():
     build_config = _read_build_config()
     return [{
-        'version': build_config['libraryVersion'],
         'name': name,
         'path': project['path'],
-        'artifactId': project['artifactId'],
-        'publications': [{
-            'name': publication['name'],
-            'type': publication['type'],
-        } for publication in project['publications']]
     } for (name, project) in build_config['projects'].items()]
 
 
+def get_version():
+    return _read_build_config()["libraryVersion"]
```

Comment:
Note: this is going to hit the memoized cached call of the buildconfig, so we're okay 👍